VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@bguertin, @BenjiLee - For now lets keep the marketing request for "Discover Courses" on the home page even though this doesn't align with other messages elsewhere. Fine to keep the button where it is as well for now and merge this. 

@roderickmorales - We can revisit the other places we say "Discover" (or "Explore" on dashboard ;) ) in a future ticket. 

Thanks!